### PR TITLE
Fix for NPE caused by Log.d in ThumbnailsCacheManager

### PR DESCRIPTION
**Pull request by ezaquarii**
Fixes #4545 

Signed-off-by: Chris Narkiewicz <hello@ezaquarii.com>

### CODE REVIEW

File: src/main/java/com/owncloud/android/datamodel/ThumbnailsCacheManager.java [hunk 1]
```diff
@@ -142,7 +143,7 @@ protected Void doInBackground(File... params) {
                         mThumbnailCache = new DiskLruImageCache(diskCacheDir, DISK_CACHE_SIZE, mCompressFormat,
                                                                 mCompressQuality);
                     } catch (Exception e) {
-                        Log_OC.d(TAG, e.getMessage());
+                        Log_OC.d(TAG, String.format(Locale.US, "Disk cache init failed: %s", e.getMessage()));
```
Comment (ezaquarii) [inline src/main/java/com/owncloud/android/datamodel/ThumbnailsCacheManager.java]: I suspect message could be `null`, causing `Log.d` to throw inside native platform implementation.

Comment (ezaquarii) [reply src/main/java/com/owncloud/android/datamodel/ThumbnailsCacheManager.java]: Actually, I managed to reproduce it by passing null to `Log.d`.

Strangely it does handle null under debugger, but crashes without debugger. :)

**Comment by AndyScherzinger** [comment]
To me it looks like might solve the matter while we probably have such an issue in more than one place within the thumbs class.

**Comment by ezaquarii** [comment]
Yeah, there is a library fix that solves this for good.

**Comment by tobiasKaminsky** [comment]
Nice, doubled preventing npe :-)
Plus we get at least here the "init failed" part.

**Pull request merged**